src/components/registration/RegistrationTable.jsx

```diff
@@ -54,12 +55,33 @@ function getTableColumns(extraFields) {
 
 function RegistrationTable({ families }) {
   const { parentFields } = useContext(FieldsContext);
+  const [open, setOpen] = useState(false);
```

Comment:
can we make the open/close names a bit more descriptive? something like `openFamilyDetail` maybe